irida_staramr_results/config/config.yml

```diff
@@ -0,0 +1,12 @@
+# -----------------------------
```

Comment:
I think this file would fit better at the top level with the name `example_config.yml` or `config_example.yml`. Our users won't be using a built in config file since their systems won't have the IP you have in the example, so labeling the file as an example file will help make that clear for anyone who wants to use the source code.

Reply:
Done.